lib/disk_store.ml

```diff
@@ -1,9 +1,9 @@
-let state_dir_root = Fpath.v @@ Filename.concat (Sys.getcwd ()) "var"
+let state_dir_root = ref (Fpath.v @@ Filename.concat (Sys.getcwd ()) "var")
```

Comment:
Perhaps this should start as `ref None`, to make sure no-one uses the default before it gets set to the correct value?

Reply:
I think it's ok to leave it as-is, `Disk_store` isn't exposed and it's pretty clear that callers in `Current` should use `state_dir`.

> to make sure no-one uses the default before it gets set to the correct value?

The tests use the default value.

Reply:
`state_dir` uses this, and therefore has the same problem, though. Making it an option would let `state_dir` fail if used too early.